dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletAPIPreHook.java

```diff
@@ -331,6 +332,23 @@ default boolean cleanField(final Structure structure, final Date deletionDate,
 	public default boolean getContentletReferences(Contentlet contentlet, User user, boolean respectFrontendRoles){
       return true;
     }
+
+	/**
+	 * This is a simplified version of the more complex {@link #getContentletReferences(Contentlet, User, boolean)}
+	 * method. This one will only be focused on querying the database to return the number of Containers that include
+	 * the specified Contentlet ID.
+	 * <p>The result provided by this method can be used to customize or determine specific behaviors. For example,
+	 * this
+	 * piece of information is used by the dotCMS UI to ask the User whether they want to edit a Contentlet referenced
+	 * everywhere, or if dotCMS should create a copy of such a Contentlet so they can edit that one version.</p>
+	 *
+	 * @param contentletId The Contentlet ID whose references will be retrieved.
+	 *
+	 * @return The number of times the specified Contentlet is added to a Container in any HTML Page.
+	 */
+	default boolean getContentletReferenceCount(final String contentletId) {
```

Comment:
the doc says it returns a number but it returns a boolean?

Reply:
Done!